modules/ROOT/pages/7.1.1-release-notes.adoc

```diff
@@ -168,6 +168,15 @@ As a result, emoji's now render in all occasion without redirecting.
 
 For more information about these changes, see xref:emoticons.adoc#emoticons_images_url[Emoticons#emoticons_images_url].
 
+==== Iframes in dialogs were not rendering rounded borders correctly.
+// #TINY-10901
+
+Previously, when opening a dialog with an `iframe` element that had a different background color than the dialog, the corners of the dialog did not round off as expected as the border radius was applied to the dialog and not the iframe.
+
+{productname} {release-version} addresses this issue by applying the border radius to the `iframe` itself, ensuring that it renders correctly.
```

Comment:
```suggestion
{productname} {release-version} addresses this issue by also applying the border radius to the `iframe` itself, ensuring that it renders correctly.
```
The original statement implies that the border radius is no longer applied to the dialog.